src/components/form/TextInput/TextInput.js

```diff
@@ -0,0 +1,19 @@
+import { StyledTextInput } from "./TextInput.styled";
+import { useInputState } from "../../../services/hooks/useInputState";
+
+export const TextInput = ({ label, field, handleForm }) => {
```

Comment:
either `TextInput` or `NumberInput` is redundant

we can just have `Input` component 
needed type input we can pass through props